spec/factories/request_factory.rb

```diff
@@ -1,7 +1,8 @@
 # frozen_string_literal: true
 
+require_relative '../rails_helper'
```

Comment:
NB: No idea why I had to change these to require_relative from require - tests were failing with require